sign-in.txt

```diff
@@ -1,7 +1,11 @@
-﻿This is an incredibly useful text document.
+﻿﻿This is an incredibly useful text document.
 //TODO: Be sure you have created a feature branch from develop 
 //TODO: Please put your name and github username AS SHOWN below ... 
-  ... then commit, pull origin develop, merge develop into your branch, push origin feature, and review your pull request
-  
-- Brian Bansenauer, github:Bansenauer-Cascadia
-- Keith Rutler, github:keithrutler
\ No newline at end of file
+  ... then commit, pull origin develop, merge develop into your branch, 
+  push origin feature, and review your pull request 
+
+- Brian Bansenauer : Bansenauer-Cascadia
+- Keith Rutler, github:keithrutler
+- Sara Ansari : Sansari246
+- Adam Kohler, github:adamkohler
+- Alex Sparks : AlexKSparks
```

Comment:
Add your name here using the format
_yourname_, github:_yourgithubname_

Reply:
Got it! I've seen different formats on the list. Thanks for clarifying! Can't believe I forgot to put my name on the last one! Kids making me crazy today!

Reply:
No worries - looks great